enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/Utils.java

```diff
@@ -66,4 +75,58 @@ public static EndpointCluster processEndpoints(org.wso2.choreo.connect.discovery
         }
         return endpointCluster;
     }
+
+    public static ResourceConfig buildResource(Operation operation, String resPath, Map<String,
+            List<String>> apiLevelSecurityList) {
+        ResourceConfig resource = new ResourceConfig();
+        resource.setPath(resPath);
+        resource.setMethod(ResourceConfig.HttpMethods.valueOf(operation.getMethod().toUpperCase()));
+        resource.setTier(operation.getTier());
+        resource.setDisableSecurity(operation.getDisableSecurity());
+        Map<String, List<String>> securityMap = new HashMap<>();
+        if (operation.getSecurityList().size() > 0) {
+            for (SecurityList securityList : operation.getSecurityList()) {
+                for (Map.Entry<String, Scopes> entry : securityList.getScopeListMap().entrySet()) {
+                    securityMap.put(entry.getKey(), new ArrayList<>());
+                    if (entry.getValue() != null && entry.getValue().getScopesList().size() > 0) {
+                        List<String> scopeList = new ArrayList<>(entry.getValue().getScopesList());
+                        securityMap.replace(entry.getKey(), scopeList);
+                    }
+                    // only supports security scheme OR combinations. Example -
+                    // Security:
+                    // - api_key: []
+                    //   oauth: [] <-- AND operation is not supported hence ignoring oauth here.
```

Comment:
Does this Security scheme means it is an `AND` operation. When it comes to swagger/openAPI it is an `OR`.
  https://swagger.io/specification/#security-requirement-object

Reply:
This is AND
```
Security:
   - api_key: []
      oauth: [] 
```

This is OR
```
Security:
   - api_key: []
   - oauth: [] 
```
We only support OR

Reply:
ack.